.drone.star

```diff
@@ -16,8 +16,8 @@ config = {
     'onlyoffice':'frontend'
   },
   'apiTests': {
-    'coreBranch': 'master',
-    'coreCommit': '27eaa32d777052d1d210647cd38895e647c48fed',
+    'coreBranch': 'changes-tusTests-invalid-file-names',
```

Comment:
We can't merge like this. We need a "general" PR to OCIS that updates the coreCommit for master and has all the needed expected-failures changes for whatever is now merged in core master.

Reply:
I will change it back to `'coreBranch': 'master'` once this PR gets merged https://github.com/owncloud/core/pull/38226